frontend/src/metabase/dashboard/components/Dashboard/Dashboard.tsx

```diff
@@ -130,10 +131,13 @@ interface DashboardProps {
     cardId: CardId;
     tabId: DashboardTabId | null;
   }) => void;
+  addHeadingDashCardToDashboard: (opts: NewDashCardOpts) => void;
+  addMarkdownDashCardToDashboard: (opts: NewDashCardOpts) => void;
+  addLinkDashCardToDashboard: (opts: NewDashCardOpts) => void;
   archiveDashboard: (id: DashboardId) => Promise<void>;
 
   onRefreshPeriodChange: (period: number | null) => void;
-  setEditingDashboard: (dashboard: IDashboard) => void;
+  setEditingDashboard: (dashboard: IDashboard | boolean) => void;
```

Comment:
I'm not sure it should accept a `boolean` at this point :(

Reply:
I also thought it was super weird, but when it's passed to the header, we pass "false" when we're done editing :/
https://github.com/metabase/metabase/blob/01fc5ae5a26799d28a5affbceab14bd44b20e1bb/frontend/src/metabase/dashboard/components/DashboardHeader/DashboardHeader.tsx#L254